### PR TITLE
Fix broadcasting finished checkpoint block + native impl for error messages

### PR DESCRIPTION
**Pull request by marcinwadon**

### CODE REVIEW

File: src/main/scala/org/constellation/util/HealthChecker.scala [hunk 1]
```diff
@@ -94,7 +94,17 @@ class HealthChecker[F[_]: Concurrent](
...
       nextSnapshotHeights <- peers.values.toList
         .map(_.peerMetadata.toPeerClientMetadata)
-        .traverse(apiClient.snapshot.getNextSnapshotHeight().run)
+        .traverse(
+          a =>
+            apiClient.snapshot
+              .getNextSnapshotHeight()
+              .run(a)
+              .map {
+                case (id, height) => (id, height)
```
Comment (TheMMaciek) [inline src/main/scala/org/constellation/util/HealthChecker.scala:103]: is that part needed?

Comment (marcinwadon) [reply src/main/scala/org/constellation/util/HealthChecker.scala:103]: Yes, in case of incorrect request, we can see it in logs on mainnet

Comment (marcinwadon) [reply src/main/scala/org/constellation/util/HealthChecker.scala:103]: Huh, wait

Comment (marcinwadon) [reply src/main/scala/org/constellation/util/HealthChecker.scala:103]: The map itself is useless, I’ll remove it

**Pull request merged**